presets/4.4/tune/karate/karate_brushless_whoop.txt

```diff
@@ -0,0 +1,162 @@
+#$ INCLUDE: presets/4.4/tune/defaults.txt
+#$ INCLUDE: presets/4.3/filters/defaults.txt
+
+# -- Gyro lowpass filters --
+set gyro_lpf1_static_hz = 0
+set gyro_lpf1_dyn_min_hz = 500
+set gyro_lpf1_dyn_max_hz = 1000
+set simplified_gyro_filter = OFF
+
+#$ OPTION BEGIN (UNCHECKED): Bosch (BMI) gyro - experimantal
+    set gyro_lpf1_dyn_min_hz = 0
+    set gyro_lpf1_dyn_max_hz = 0
+#$ OPTION END
+
+# -- Gyro Dynamic Notches --
+set dyn_notch_count = 2
+set dyn_notch_q = 600
+set dyn_notch_max_hz = 650
+
+# -- Dterm filtering --
+set dterm_lpf1_dyn_expo = 7
+
+# -- RPM filtering --
+set dshot_bidir = ON
+set rpm_filter_fade_range_hz = 100
+
+# -- Misc --
+set thrust_linear = 20
+
+
+# -- PIDsum limits --
+set iterm_limit = 500
+set pidsum_limit_yaw = 1000
+
+
+# -- PID values --
+set p_pitch = 79
+set i_pitch = 71
+set d_pitch = 45
+set f_pitch = 127
+set p_roll = 76
+set i_roll = 68
+set d_roll = 43
+set f_roll = 122
+set p_yaw = 76
+set i_yaw = 68
+set f_yaw = 122
+set d_min_pitch = 31
+set simplified_master_multiplier = 170
+set simplified_i_gain = 50
+set simplified_d_gain = 60
+set simplified_dmax_gain = 130
+set simplified_feedforward_gain = 60
+set simplified_pitch_d_gain = 90
+
+#$ OPTION BEGIN (UNCHECKED): Dshot600
+    set dshot_bidir = ON
+    set motor_pwm_protocol = DSHOT600
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Dshot300
+    set dshot_bidir = ON
+    set motor_pwm_protocol = Dshot300
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Set 12 poles for whoop motors
+	set motor_poles = 12
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): Dynamic idle for 11000kv 2s
+    set dyn_idle_min_rpm = 150
+    set dyn_idle_p_gain = 35
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Vbat sag compensation
+	set vbat_sag_compensation = 100
+#$ OPTION END	
+
+#$ OPTION_GROUP BEGIN: Some popular RC Links
+
+    #$ OPTION BEGIN (UNCHECKED): Tracer/ELRS 250Hz
+        # Tracer/ELRS 250Hz
+        #$ INCLUDE: presets/4.3/rc_link/defaults.txt
+        feature RX_SERIAL
+        set serialrx_provider = CRSF
+
+        set feedforward_averaging = 2_POINT
+        set feedforward_smooth_factor = 35
+        set feedforward_jitter_factor = 4
+        set feedforward_boost = 18
+
+        set rc_smoothing_auto_factor = 25
+        set rc_smoothing_auto_factor_throttle = 25
+    #$ OPTION END
+
+    #$ OPTION BEGIN (UNCHECKED): ELRS 500Hz
+        # ERLS 500Hz
+        #$ INCLUDE: presets/4.3/rc_link/defaults.txt
+        feature RX_SERIAL
+        set serialrx_provider = CRSF
+
+        set feedforward_averaging = 2_POINT
+        set feedforward_smooth_factor = 65
+        set feedforward_jitter_factor = 3
+        set feedforward_boost = 18
+
+        set rc_smoothing_auto_factor = 25
+        set rc_smoothing_auto_factor_throttle = 25
+    #$ OPTION END
+#$ OPTION_GROUP END
+
+#$ OPTION_GROUP BEGIN: Optional rates
+
+    #$ OPTION BEGIN (UNCHECKED): sugarK's whoop rates
+        set thr_expo = 80
+		set roll_expo = 54
```

Comment:
plz replace tabs with spaces here too in this section

Reply:
your spacing doest play nice with my editor.. will work out how to fix

Reply:
> your spacing doest play nice with my editor.. will work out how to fix

in most places you got it good :)

Reply:
Da